src/Broadway/Bundle/BroadwayBundle/DependencyInjection/RegisterSagaCompilerPass.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Broadway\Bundle\BroadwayBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+class RegisterSagaCompilerPass implements CompilerPassInterface
+{
+    private $multipleSagaManagerService;
+    private $tagName;
+
+    public function __construct($multipleSagaManagerService, $tagName)
```

Comment:
Could add a docblock here?
For arguments that don't have a type hint, we like to have a docblock.